site/public/css/electronic.css

```diff
@@ -27,7 +27,7 @@ main#main.full-screen-mode .content {
 
 #wrapper {
     padding-top: 30px;
-    min-height: 0;
```

Comment:
Question: Is this min-height in this file needed ? The `global.css` will put the `#wrapper` CSS rule on every page with the `min-height: max-content;`. 

When I see it on firefox it seems the duplicate rules get ignored and the others are applied. Not sure if this is the same on other browsers. Would just removing the `min-height` in the electronic.css cause any issues instead?

Reply:
@Eceptonsu @shailpatels  I also though that global.css must do the work but ..
![image](https://user-images.githubusercontent.com/92226302/171699754-b3904d77-9549-456a-b0ae-ef7ccd22c7c0.png)
Here in the screenshot (firefox browser) , It derives the css component from electronic.css and without it , the property didn't worked.
Hence, addition there was done.